src/frontend/deployment_details.go

```diff
@@ -50,18 +60,14 @@ func loadDeploymentDetails(httpRequest *http.Request) map[string]string {
 		"hostname": podHostname,
 	}).Debug("Loaded deployment details")
 
-	detailsLoadingStatus = DetailsAreLoaded
+	areDeploymentDetailsLoaded = true
 
 	return deploymentDetailsMap
 }
 
 func getDeploymentDetailsIfLoaded(httpRequest *http.Request) map[string]string {
```

Comment:
This is completely redundant if we change the `deploymentDetailsMap` to be `nil` at initialization (so remove the `make()` from there, so this entire method can go away.

Reply:
I've preserved the `getDeploymentDetailsIfLoaded()` because it implies that the thing being returned might not have been loaded; thus, I assume anyone using/seeing `getDeploymentDetailsIfLoaded()` will cautiously check if it returns `nil`.
Thoughts?